kong/pdk/request.lua

```diff
@@ -841,6 +841,51 @@ local function new(self)
     return ngx.ctx.KONG_PROCESSING_START or (req.start_time() * 1000)
   end
 
+  local EMPTY = {}
+
+  local function capture_wrap(capture)
+    local named_captures = {}
+    local unnamed_captures = {}
+    local wrapped_captures = {
+      unnamed = unnamed_captures,
+      named = named_captures,
+    }
+    for k, v in pairs(capture) do
+      if type(k) == "number" then
+        unnamed_captures[k] = v
+  
+      elseif type(k) == "string" then
+        named_captures[k] = v
+  
+      else
+        kong.log.err("unknown capture key type: ", k)
+      end
+    end
+  
+    return wrapped_captures
```

Comment:
We can simply return a `{named = ..., unamed = ...}` then remove var `wrapped_captures`